components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/SelfRegisterApi.java

```diff
@@ -240,4 +226,16 @@ public void validateCodePostCall(CodeValidationRequest code) throws ApiException
         };
         apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames, localVarReturnType);
     }
+
+    private void setBasePath(String tenantDomain) {
+
+        if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled()) {
+            basePath = IdentityManagementEndpointUtil.buildTenantQualifiedUrl(
+                    IdentityManagementEndpointConstants.UserInfoRecovery.USER_API_RELATIVE_PATH);
+        } else if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equalsIgnoreCase(tenantDomain)) {
+            basePath = IdentityManagementEndpointUtil.buildEndpointUrl(
+                    "t/" + tenantDomain + IdentityManagementEndpointConstants.UserInfoRecovery.USER_API_RELATIVE_PATH);
```

Comment:
Cant't we make  "t/" as a constant? Since it s been used in several places.

Reply:
This method has been removed.